cypress/fixtures/registrationPasswordMissmatch.json

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
misspelling in the file name:
"mismatch" is one S

Reply:
Fixed

Reply:
```
registrationPasswordMissmatch.json
```
this is what I was referring to